feat : add-common-films

MaXM91 opened the pull request:
feat : 
- ввести вывод общих фильмов по ид 2 пользователь.
    Добавить методы:
    - getSharedFilms в FilmController.java,
    - getFilmsShared в FilmService.java,
    - foundFilmsShared в интерфейс FilmRepository.java,
    - foundFilmsShared в FilmRepositoryImpl.java,
    - foundFilmsShared в InMemoryFilmRepository.java.
- провести тестирование методов.
    Добавить метод:
    - shouldGetFilmsSharedTest() в FilmRepositoryTests.java.

pesotskypv reviewed (approved):
Критических замечаний нет. Принято.

pesotskypv committed:
feat :
- ввести вывод общих фильмов по ид 2 пользователь.
    Добавить методы:
    - getSharedFilms в FilmController.java,
    - getFilmsShared в FilmService.java,
    - foundFilmsShared в интерфейс FilmRepository.java,
    - foundFilmsShared в FilmRepositoryImpl.java,
    - foundFilmsShared в InMemoryFilmRepository.java.
- провести тестирование методов.
    Добавить метод:
    - shouldGetFilmsSharedTest() в FilmRepositoryTests.java.
